src/ResourceManager/Network/Commands.Network/help/Get-AzureRMNetworkWatcherReachabilityReport.md

```diff
@@ -0,0 +1,297 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRMNetworkWatcherReachabilityReport
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+### SetByName (Default)
+```
+Get-AzureRMNetworkWatcherReachabilityReport -NetworkWatcherName <String> -ResourceGroupName <String>
+ [-Provider <System.Collections.Generic.List`1[System.String]>]
+ [-Location <System.Collections.Generic.List`1[System.String]>] -StartTime <DateTime> -EndTime <DateTime>
+ [-Country <String>] [-State <String>] [-City <String>] [-DefaultProfile <IAzureContextContainer>]
+```
+
+### SetByResource
+```
+Get-AzureRMNetworkWatcherReachabilityReport -NetworkWatcher <PSNetworkWatcher>
+ [-Provider <System.Collections.Generic.List`1[System.String]>]
+ [-Location <System.Collections.Generic.List`1[System.String]>] -StartTime <DateTime> -EndTime <DateTime>
+ [-Country <String>] [-State <String>] [-City <String>] [-DefaultProfile <IAzureContextContainer>]
+```
+
+### SetByResourceId
+```
+Get-AzureRMNetworkWatcherReachabilityReport -ResourceId <String>
+ [-Provider <System.Collections.Generic.List`1[System.String]>]
+ [-Location <System.Collections.Generic.List`1[System.String]>] -StartTime <DateTime> -EndTime <DateTime>
+ [-Country <String>] [-State <String>] [-City <String>] [-DefaultProfile <IAzureContextContainer>]
+```
+
+## DESCRIPTION
+The Get-AzureRmNetworkWatcherReachabilityReport gets the relative latency score for internet service providers from a specified location to Azure regions.
+
+## EXAMPLES
+
+### Example 1
+```
+$nw = Get-AzureRmNetworkWatcher -Name NetworkWatcher -ResourceGroupName NetworkWatcherRG
+Get-AzureRmNetworkWatcherReachabilityReport -NetworkWatcher $nw -Location "West US" -Country "United States" -StartTime "2017-10-10" -EndTime "2017-10-12"
+
+"aggregationLevel" : "Country",
+"providerLocation" : {
+    "country" : "United States"
+},
+"reachabilityReport" : [
+    {
+        "provider" : "Frontier Communications of America, Inc. - ASN 5650",
+        "azureLocation": "West US",
+        "latencies": [
+            {
+                "timeStamp": "2017-10-10T00:00:00Z",
+                "score": 94
+            },
+            {
+                "timeStamp": "2017-10-11T00:00:00Z",
+                "score": 94
+            },
+            {
+                "timeStamp": "2017-10-12T00:00:00Z",
+                "score": 94    
+            }
+        ]  
+    }
+]
+
```

Comment:
@irrogozh it looks like you're missing ending ` ``` ` in this example

Reply:
Fixed